examples/custom-path/gatsby-config.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  siteMetadata: {
+    title: `Gatsby Default Starter`,
+    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
+    author: `@gatsbyjs`,
+  },
+  plugins: [
+    {
+      resolve: `@prismicio/gatsby-source-prismic-graphql`,
+      options: {
+        repositoryName: 'gatsby-demo-site',
+        pages: [
+          {
+            type: 'Blogpos',
+            customPath: node => {
+              return `/${node.title[0].text}/${node._meta.uid}`;
+            },
+            extraPageFields: 'title',
```

Comment:
@MarcMcIntosh here is the use case.

This one is contrived, but in my real use case I have a field `post_date` on each blog post, and want the url to be `/blog/:year/:month/:day/:uid`

At first I tried using `extraPageFields` at the plugin options level, but that is very limiting because it means _every_ custom type must contain the specified field.

See the changes elsewhere that support `extraPageFields` at page level (where I think it belonged in the first place).